### PR TITLE
fix: some idiomatic changes

### PR DESCRIPTION
**Pull request by carlank**

### CODE REVIEW

File: src/Bot.js [hunk 1]
```diff
@@ -57,12 +50,20 @@ class Bot {
...
         });
     }
 
+    /**
+     * Configure a new source
+     * @param {StaticSource} source
+     */
+    addSource(source) {
+        this.sources.push(source);
+    }
```
Comment (Stratadox) [inline src/Bot.js:65]: Why move it?

File: src/Bot.js [hunk 2]
```diff
@@ -26,27 +26,20 @@ class Bot {
      * @param  {Array}    tags     Tags to considered relevant
      * @throws When refused
      */
-    configure(channel, delay, callback, tags = []) {
+    configureChannel(channel, delay, callback, tags = []) {
```
Comment (Stratadox) [inline src/Bot.js:29]: :ok_hand:

File: src/Bot.js [hunk 3]
```diff
@@ -71,8 +72,7 @@ class Bot {
      * @todo add weighted random, where weight is # of tags in common
      */
     chooseSourceFor(channel) {
-        for (let i in this.sources) {
-            let source = this.sources[i];
+        for (const source of this.sources) {
```
Comment (Stratadox) [inline src/Bot.js:75]: Ah nice, that's what I was looking for thx

File: src/Bot.js (hunk 1 above)
Comment (carlank) [reply src/Bot.js:65]: Grouping source-related methods together

Comment (Stratadox) [reply src/Bot.js:65]: Hehe I was grouping configuration methods together

**Pull request merged**